### DISCUSSION TITLE
GitHub repo for handlebarsjs.com

### DISCUSSION THREAD

**Discussion by samatdav** [handlebars-lang/handlebars.js#2014] [Q&A]
Hello!

I am excited about using Handlebars and really like the the demo at https://handlebarsjs.com/playground.html
Is there a source for this page to see how it's setup? Or is it private? Sorry, couldn't easily find it. 

Thank you!

**Comment by jaylinski** [comment]
In case someone else has the same question, here is the source: https://github.com/handlebars-lang/docs
